cli/command/trust/helpers.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/docker/notary/tuf/data"
 )
 
+const releasedRoleName = "Repo Admin"
```

Comment:
Can you also use this constant in `formattedSigners = append(formattedSigners, "(Repo Admin)")` in inspect.go's line 200?